.github/workflows/e2etests.yaml

```diff
@@ -125,3 +124,116 @@ jobs:
           commit_author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
           commit_options: '--signoff'
           commit_message: '[Docs] Test status of adapter'
+
+  SetPatternfileForServicemeshAndSampleApp:
+    runs-on: ubuntu-latest
+    outputs:
+      sm_version: ${{ steps.gettag.outputs.release }}
+      adapter_version:  ${{ env.version }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Get version of adapter
+        run: |
+            if [ ${{ github.event_name }} == "release" ];then
+                echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+            else 
+                echo "version=edge" >> $GITHUB_ENV
+            fi
+      - name: Get latest release tag
+        id: gettag
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          repository: linkerd/linkerd2
+          excludes: prerelease, draft, edge 
+      - name: Change service mesh version in patternfile
+        run: | 
+            yq e -i '.services.linkerdinstallation.settings.linkerdVersion="${{ steps.gettag.outputs.release }}"' ./.github/install/deployWithSampleApp.yaml 
+            cat ./.github/install/deployWithSampleApp.yaml
+      - name: Uploading file
+        uses: actions/upload-artifact@v2
+        with:
+          name: patternfile
+          path: ./.github/install/deployWithSampleApp.yaml 
+
+  TestLinkerdForServicemeshAndSampleApp:
+    needs: SetPatternfileForServicemeshAndSampleApp
+    uses: meshery/meshery/.github/workflows/test_adapters.yaml@master
```

Comment:
@Zanda256 Will you change this to refer to the v2 test_adapters that was created?